index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>StartApp</title>
-</head>
-<body>
+  </head>
+  <body>
     <h1>Million Dollar Page!</h1>
-</body>
-</html>
\ No newline at end of file
+    <h2>New changs from Aigerim</h2>
```

Comment:
This text looks wrong, please fix your mistake!

Reply:
Ok I will try

Reply:
I fixed it